2-mandatory/1-weather-report.js

```diff
@@ -12,52 +12,53 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+  // TODO
+  let stringOutput = [];
+  for (let i = 0; i < cities.length; i++) {
+    stringOutput.push(
+      `The temperature in ${cities[i]} is ${temperatureService(
+        cities[i]
+      )} degrees`
```

Comment:
We usually keep the function body and arguments on the same line, unless running out of space.

Reply:
I think it wasn't @HThanki fault, this indentation happens when we save the code in VSCode.